### ISSUE TITLE
dilbert.com moved, looking for stable source of images, app cannot work right now

### ISSUE THREAD

**Issue by JonnyTech** [smarek/Simple-Dilbert#63]
From Scott Adams Twitter feed https://twitter.com/ScottAdamsSays/status/1635200015886077953 ([nitter](https://nitter.net/ScottAdamsSays/status/1635200015886077953))

"Dilbert Reborn" launches March 13 on [http://Scottadams.locals.com](https://t.co/hhnnMv9FGZ) exclusively. (Subscription)

**Comment by smarek** [comment]
Hi Jonny, this seems like end of the story. Comic is currently published under paywall (subscription) and old strips are not hosted on dilbert.com. 
Do you think, we have any alternative to get stable stream of dilbert comics (and archives) now?

**Comment by JonnyTech** [comment]
Indeed, this does seem like the end. Thank you for your app, it was much appreciated.

There are some archives on the wayback machine and a Dilbert Reborn reddit feed (https://www.reddit.com/r/dilbert/comments/11qdh9a/i_signed_up_so_you_didnt_have_to/) but not sure how long they will remain.

**Comment by smarek** [comment]
Yeah, those are not stable sources I could rely on. I will keep this ticket open now, since the situation is developing rapidly, and we might get reliable source of images later on, til then, I'll put-up notification for everybody to know.

**Comment by mnalis** [comment]
Could we at least have release that allows access to old strips? e.g. If I go to `Mar 2, 2018` it says `Dilbert.com is not available. Loading failed` and strip is not displayed and also `Save current image` fails, of course.

But if I click on `Open in Browser` it leads to https://assets.amuniversal.com/1c5f4a60f24f0135172e005056a9545d it **shows the picture just fine**. Could it integrated so people could still see / save the strips they are interested in the SimpleDilbert app?

**Comment by smarek** [comment]
@mnalis 

1) please test with data turned off, it could load the data from cache just fine under such conditions
2) saving the image should work, what error are you getting?
3) yes, the url is just fine, i don't know, why previous 2 issues happen for you, can you try and provide adb logcat logs ? found some guide, might be helpful: https://www.xda-developers.com/guide-sending-a-logcat-to-help-debug-your-favorite-app/
4) i'm not very motivated to do anything since the dlibert is dead, sorry, hopefully official archives will re-appear (you can query Scott directly) and then it will be meaningful to release new version. if you can develop it, do a PR and i'll tackle that asap
5) i believe AMMU will delete those files as well soon, these cached URLs will stop working as well

**Comment by mnalis** [comment]
Thanks @smarek. I've put the answers in new issue https://github.com/smarek/Simple-Dilbert/issues/65 in order not to pollute this one.  It seems the app is not following `http` -> `https` redirects so fetching the existing older comics fail.

While I hope you'll find a little motivation to help users of the app to save their favourites, I'll understand if you're no longer interested and decide to archive this repo. If one is sufficiently motivated, I guess one can always download the [Dilbert archives](https://www.reddit.com/r/DataHoarder/comments/11rtliq/complete_dilbert_comic_archive/) and manually find dates and extract comics they liked one by one.

Thanks for the ride in any case! :heart:
